NuGet.Docs/Create/TargetFrameworks.md

```diff
@@ -0,0 +1,647 @@
+<td><i>dotnet5.1</i><br/>netstandard1.0</td>
+</tr>
+<tr>
+<td>Profile328</td>
+<td>
+.NETFramework 4.0<br/>
+Silverlight 5.0<br/>
+Windows 8.0<br/>
+WindowsPhone 8.1<br/>
+WindowsPhone 8.0</td>
+<td>portable-net40+sl5+win8+wpa81+wp8</td>
+<td></td>
+</tr>
+<tr>
+<td>Profile336</td>
+<td>
+.NETFramework 4.0.3<br/>
+Silverlight 5.0<br/>
+Windows 8.0<br/>
+WindowsPhone 8.1<br/>
+WindowsPhone 8.0</td>
+<td>portable-net403+sl5+win8+wpa81+wp8</td>
+<td></td>
+</tr>
+<tr>
+<td>Profile344</td>
+<td>
+.NETFramework 4.5<br/>
+Silverlight 5.0<br/>
+Windows 8.0<br/>
+WindowsPhone 8.1<br/>
+WindowsPhone 8.0</td>
+<td>portable-net45+sl5+win8+wpa81+wp8</td>
+<td></td>
+</tr>
+</table>
+
+Additionally, there are optional frameworks provided by Xamarin that are supported by the official NuGet clients.  More details about these frameworks are available from [Xamarin](http://www.xamarin.com).
+ 
+<table class="reference">
+  <tr>
+    <th>Name</th>
+    <th>Description</th>
+    <th>.NET Platform Standard</th>
+  </tr>
+  <tr>
+    <td>monoandroid</td>
+    <td>Mono Support for Android OS</td>
+    <td><i>dotnet5.5</i><br/>netstandard1.4</td>
+  </tr>
+  <tr>
+    <td>monotouch</td>
+    <td>Mono Support for iOS</td>
+    <td><i>dotnet5.5</i><br/>netstandard1.4</td>
+  </tr>
+  <tr>
+    <td>monomac</td>
+    <td>Mono Support for OSX</td>
+    <td><i>dotnet5.5</i><br/>netstandard1.4</td>
+  </tr>
+  <tr>
+    <td>xamarinios</td>
+    <td>Support for Xamarin for iOS</td>
+    <td><i>dotnet5.5</i><br/>netstandard1.4</td>
+  </tr>
+  <tr>
+    <td>xamarinmac</td>
+    <td>Supports for Xamarin for Mac</td>
+    <td><i>dotnet5.5</i><br/>netstandard1.4</td>
+  </tr>
+  <tr>
+    <td>xamarinpsthree</td>
+    <td>Support for Xamarin on Playstation 3</td>
+    <td><i>dotnet5.5</i><br/>netstandard1.4</td>
+  </tr>
+  <tr>
+    <td>xamarinpsfour</td>
+    <td>Support for Xamarin on Playstation 4</td>
+    <td><i>dotnet5.5</i><br/>netstandard1.4</td>
+  </tr>
+  <tr>
+    <td>xamarinpsvita</td>
+    <td>Support for Xamarin on PS Vita</td>
+    <td><i>dotnet5.5</i><br/>netstandard1.4</td>
+  </tr>
+  <tr>
+    <td>xamarinwatchos</td>
+    <td>Xamarin for Watch OS</td>
+    <td><i>dotnet5.5</i><br/>netstandard1.4</td>
+  </tr>
+  <tr>
+    <td>xamarintvos</td>
+    <td>Xamarin for TV OS</td>
+    <td><i>dotnet5.5</i><br/>netstandard1.4</td>
+  </tr>
+  <tr>
+    <td>xamarinxboxthreesixty</td>
+    <td>Xamarin for XBox 360</td>
+    <td><i>dotnet5.5</i><br/>netstandard1.4</td>
+  </tr>
+  <tr>
+    <td>xamarinxboxone</td>
+    <td>Xamarin for XBox One</td>
+    <td><i>dotnet5.5</i><br/>netstandard1.4</td>
+  </tr>
+</table>
+
+Stephen Cleary has a blog post that demonstrates a tool that will mine the list of supported portable class libraries on a workstation and report the features of them: [http://blog.stephencleary.com/2012/05/framework-profiles-in-net.html](http://blog.stephencleary.com/2012/05/framework-profiles-in-net.html)
+
+
+### .NET Platform Standard 
```

Comment:
Explain that this isn't part of 3.3 the new netstandard tfm will be coming in 3.4